examples/01.Basics/Skeleton_Multi/Skeleton_Multi_Unit2.cpp

```diff
@@ -1,4 +1,6 @@
-#include <MozziGuts.h>        // This file, too, will have to include the Mozzi headers.
+#include <MozziHeadersOnly.h>        // <Mozzi.h> should be included only once in the whole program. Sketches needing
+                                     // core Mozzi functions in more than one .cpp file, shall include MozziHeadersOnly.h
+                                     // in all but one.
```

Comment:
That answers my previous question. I think this deserves a comment in the Learn of FAQ of the website at some point.

Reply:
Indeed. Adding a task item in the documentation PR, and "resolving" it, here.